### PR TITLE
Knauer US-11518: Types For Timber Design - MoistureClass and ServiceCondition

### PR DESCRIPTION
**Pull request by aliciaknauer**
# Description

The two functions set_timber_moisture_class and set_timber_service_conditions are implemented here. Since there are some issues with setting the right code, a thorough instruction can be found in every test file on how to test. Because of this reason the pytest.skipif function was added, so it doesn't interfere with existing tests. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Unit Tests
- [ ] Attached examples

**Test Configuration**:
* RFEM / RSTAB version: 6.02.0060
* Python version: 3.11

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: RFEM/TypesForTimberDesign/timberMoistureClass.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                params: dict = None):
+        """
+        Args:
+            no (int): Timber Moisture Class Tag
+            name (str): User Defined Moisture Class Name
+            members (str): Assigned Members
+            member_sets (str): Assigned Member Sets
+            surfaces (str): Assigned Surfaces
+            surface_sets (str): Assigned Surface Sets
+            moisture_class (enum): Timber Moisture Class
```
Comment (heetrojivadiya) [inline RFEM/TypesForTimberDesign/timberMoistureClass.py:24]: Description should be Timber Moisture Class Type Enumeration

File: RFEM/TypesForTimberDesign/timberServiceCondition.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        """
+        Args:
+            no (int): Timber Service Conditions Tag
+            name (str): User Defined Timber Service Condition Name
+            members (str): Assigned Members
+            member_sets (str): Assigned Member Sets
+            surfaces (str): Assigned Surfaces
+            surface_sets (str): Assigned Surface Sets
+            standard (int): Code Number
+            moisture_service_condition (enum): Timber Service Condition
```
Comment (heetrojivadiya) [inline RFEM/TypesForTimberDesign/timberServiceCondition.py:32]: description should be Timber Moisture Service Condition Type Enumeration

File: RFEM/enums.py [hunk 3]
```diff
@@ -2472,3 +2472,28 @@ class NoteSurfaceReferenceType(Enum):
     Note Surface Reference Type Enumeration
     '''
     OFFSET_TYPE_XY, OFFSET_TYPE_XZ, OFFSET_TYPE_YZ = range(3)
+
+class TimberMoistureClassMoistureClass(Enum):
```
Comment (heetrojivadiya) [inline RFEM/enums.py:2476]: TimberMoistureClassType is better class name

File: RFEM/enums.py [hunk 4]
```diff
@@ -2472,3 +2472,28 @@ class NoteSurfaceReferenceType(Enum):
...
+    '''
+    Timber Moisture Class
+    '''
+    TIMBER_MOISTURE_CLASS_TYPE_1, TIMBER_MOISTURE_CLASS_TYPE_2, TIMBER_MOISTURE_CLASS_TYPE_3 = range(3)
+
+class TimberServiceConditionsMoistureServiceCondition(Enum):
```
Comment (heetrojivadiya) [inline RFEM/enums.py:2482]: TimberMoistureClassType is better class name

File: RFEM/TypesForTimberDesign/timberServiceCondition.py [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+            temperature (enum): Timber Service Conditions Temperature
```
Comment (heetrojivadiya) [inline RFEM/TypesForTimberDesign/timberServiceCondition.py:33]: it would be good to add Enumeration at the end of description for variable type (enum)

**Review by OndraMichal — CHANGES_REQUESTED** [review]
Hi,
please look at branch MichalOndrej-timberServicesAndMoisture and carefully compare the tests. There is a way to execute the tests properly even if the setters are not in the WS.

**Pull request merged**